### ISSUE TITLE
torch.sparse has no attribute softmax

### ISSUE THREAD

**Issue by InstantWindy** [pytorch/pytorch#82716]
pytorch version : 1.4.0 

torch.sparse has no attribute softmax

**Comment by ngimel** [comment]
It was added after 1.4
